### ISSUE TITLE
Bug report - [YOUR BUG TITLE]

### ISSUE THREAD

**Issue by phone-burner** [CypherpunkArmory/UserLAnd#1039]
# Describe the bug

I want to use Alpine in UserLAnd,but I failed to download Alpine's rootfs.I tried to fix this problem,but I didn't find the download link.The APP stuck at'Downloading rootfs'.I cannot start Ubuntu or Debian,either.I know that reinstall UserLAnd will work,but I have about 20GB data in it and making a backup file seems impossible.How to protect my data?
# Steps to reproduce the behavior:

A list of **exact** steps to reproduce the behavior. If we cannot follow the steps to recreate the issue, we won't be able
to help you.

1. Click 'Alpine' and type in the username&password

2. wait,and then the downloader(the default system downloader) will go wrong 

3. wait,then UserLAnd will print error message

4. your UserLAnd won't work again.unless you reinstall it

# Screenshots

N/A
# Device Information

Device: [e.g. Pixel XL 2].  MI 6X
Android Version: [e.g. Oreo, 8.0].  9.0
UserLAnd Version [e.g. 0.1.2].  2.6.2
Whether the device is rooted.  no
Whether the device is running LineageOS no

Please add any other context you think might help us resolve your issue.

**Comment by corbinlc** [comment]
@MatthewTighe does the app remember if it is still waiting for a download to complete, even if you force the app to stop and reopen it?  Is there a way to tell the statemachine to reset if that is the case?

**Comment by MatthewTighe** [comment]
Yeah, but typically the download manager informs us that the download has been completed. Asset requirements are generated by the files being present, so it should just try to redownload them if it failed and for whatever reason the download manager doesn't sync up correctly

**Comment by corbinlc** [comment]
I guess I can try to replicate by killing my wifi in the middle of the download or just turning the device off or something like that.

**Comment by phone-burner** [comment]
@corbinlc My download manager said the download link was inaccessible.
